pkg/testutil/compose.go

```diff
@@ -50,15 +50,15 @@ func (cd *ComposeDir) CleanUp() {
 	os.RemoveAll(cd.dir)
 }
 
-func NewComposeDir(t testing.TB, dockerComposeYAML string) *ComposeDir {
+func NewComposeDir(t testing.TB, dockerComposeYAML string, yamlFileName string) *ComposeDir {
```

Comment:
I don't think we need this change

Reply:
Because I want to test two different compose file as the same file. I think change this method signature will help me to reuse the code as most as I can

Reply:
The filename  (`docker-compose.yaml`) can be still same as long as we use different directories?

Reply:
Ops, I have not seen the tmp file system has been used here

Reply:
Done